app/views/defence_requests/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= flash_messages %>
 
-<h1><%= "#{t(current_user.role)}  #{t('dashboard')}" %></h1>
+<h1><%= "#{t(current_user.roles.first.to_s)}  #{t('dashboard')}" %></h1>
```

Comment:
Crying out for a helper method.

Also - would roles always have a guaranteed sort order? If it doesn't, the role displayed could vary from page to page.